### ISSUE TITLE
Softmax layer implementation

### ISSUE THREAD

**Issue by SzilvasiPeter** [renan-cunha/BatchNormalization#1]
https://github.com/renan-cunha/BatchNormalization/blob/971e0189e1ecee38177cc28015b2b782ecaf56d0/src/feed_forward/layers.py#L78

We need to calculate the exponential of all elements in the input array at the softmax function.
